datatable/java/datatable/src/test/java/io/cucumber/datatable/DataTableTypeRegistryTest.java

```diff
@@ -120,4 +120,24 @@ public void returns_entry_data_table_type() {
 
         assertSame(entry, lookupTableTypeByType);
     }
+
+    @Test
+    public void parse_decimal_with_english_locale() {
+        parse_decimal_with_provided_locale(registry, "12.45", 12.45);
+    }
+
+    @Test
+    public void parse_decimal_with_polish_locale() {
+        DataTableTypeRegistry plRegistry = new DataTableTypeRegistry(Locale.forLanguageTag("pl"));
+        parse_decimal_with_provided_locale(plRegistry, "12,45", 12.45);
+
+    }
+
+    private void parse_decimal_with_provided_locale(DataTableTypeRegistry typeRegistry, String decimalString, double expectedDecimalValue) {
```

Comment:
This is an assertion method. Could you rename it to describe what it asserts, preferrably prefixing with `assert` or `expect`?